docs/about/glossary.md

```diff
@@ -101,3 +107,11 @@ A transient state is a state that only has [eventless transitions](#eventless-tr
 ## Transition
 
 A transition is a description of which target [state(s)](#state) and [actions](#action) a state machine will immediately be in when a specific [event](#event) is taken in the transition's source state.
+
+## Visual formalism
+
+A visual formalism is an exact language (like a programming language) that primarily uses visual symbols (states, transitions, etc.) instead of only code or text. State machines and statecharts are visual formalisms.
```

Comment:
```suggestion
A visual formalism is a mathematically rigorous language (like a programming language) that primarily uses visual symbols (states, transitions, etc.) instead of only code or text. State diagrams and statecharts are visual formalisms.
```